docs/source/constants.rst

```diff
@@ -0,0 +1,107 @@
+Manim Constants
+===============
+
+The ``constants.py`` under ``manimlib/`` contains variables that are used
+during setup and running manim. Some variables are not documented here as they are
+only used internally by manim.
+
+Directories
+-----------
+
+    MEDIA_DIR
+              The root directory for media files. First the environment variable
+              ``MEDIA_DIR`` is queried. If the env is not found, use the path
+              as specified in ``manimlib/media_dir.txt``. If media_dir.txt is empty,
+              a directory is created under users home. If ``MEDIA_DIR`` is not a
+              directory, ``media`` will be created under current working directory.
+              Defaults to ``manimlib/media``
+    VIDEO_DIR
+              Created under ``MEDIA_DIR``. Used to store rendered manim scenes.
+              When scene finished rendering, it will be stored under ``media/project_name/quality/scene_name.mp4``
+    RASTER_IMAGE_DIR
+              Created under ``media/designs/raster_images``. For storing
+              raster images like jpg, png, etc.
+    SVG_IMAGE_DIR
+              Created under ``media/designs/svg_images``. For storing svg files.
+    SOUND_DIR
+              Created under ``media/designs/sounds``. For storing sound files.
+    THIS_DIR
+              Path to the directory which ``constants.py`` resides.
+    FILE_DIR
+              The root directory for misc files. Specified by the environment variable
+              ``FILE_DIR``. If the variable is not found, a directory ``files`` will be
+              created under current working directory.
+    TEX_DIR
+              Created under ``FILE_DIR/files``. TexMObject and TextMobject are rendered
+              here for temporary storage.
+
+Those directories are created if they don't exist.
+
+.. note::
+    ``MOBJECT_DIR`` and ``IMAGE_MOBJECT_DIR`` is deprecated.
+
+Tex
+---
+    TEX_USE_CTEX
+              A boolean value. Change it to True if you need to use Chinese typesetting.
+    TEX_TEXT_TO_REPLACE
+              ????????
```

Comment:
This is the placeholder text in the tex template that Manim will replace when writing files[[1](https://github.com/3b1b/manim/blob/c42ae8a55bb2f1ef8259a719f17eaf519c46f3f8/manimlib/tex_template.tex#L24)][[2](https://github.com/3b1b/manim/blob/9b2c2c4784a9d619fed0b02d4af4555f12c1741c/manimlib/utils/tex_file_writing.py#L32-L34)].